test/audio/MidiFileGenerator.test.ts

```diff
@@ -140,7 +140,7 @@ describe('MidiFileGeneratorTest', () => {
             new TrackEndEvent(3840, 0) // 3840 = end of bar
         ];
         for (let i: number = 0; i < handler.midiEvents.length; i++) {
-            Logger.info('Test', `i[${i}] ${handler.midiEvents[i]}`);
+            // Logger.info('Test', `i[${i}] ${handler.midiEvents[i]}`);
```

Comment:
I am not a fan of commented code. Either we should fully remove it or if we want to keep it still for "some" purpose, a comment should explain why the commented code is kept.